### ISSUE TITLE
Can simplex-chat work without an internet connection?

### ISSUE THREAD

**Issue by 3N37** [simplex-chat/simplexmq#1282]
is it possible to run simplex-chat via a server on a lan, so as to allow chat between users on a lan, in the absence of an internet connection? and how is it done?

**Comment by DarkmatterUAE** [comment]
Yes, you can initialize the server with IP address in LAN and use them to configure your clients. Just remember to assign a static IP to the server.

**Comment by 3N37** [comment]
> Yes, you can initialize the server with IP address in LAN and use them to configure your clients. Just remember to assign a static IP to the server.


do you know where to find a guide? I tried but it doesn't work.

**Comment by DarkmatterUAE** [comment]
> is it correct to put the ip address of my internal server on the lan e.g. 192.168.0.200
>

Yes.

**Comment by 3N37** [comment]
> @3N37 were you able to get something running? I'm getting really wonky notification behaviour on my LAN setup.

I haven't managed to get it working yet, next weekend I will try harder. what problems have you had?

**Comment by faezs** [comment]
I was having issues with android not sending or recieving any messages at all. Fixed it by spoofing android's network status check server with dnsmasq so simplex thinks it online now. Everything works!
